src/Uuid.php

```diff
@@ -3,9 +3,13 @@
 namespace Kblais\Uuid;
 
 use Ramsey\Uuid\Uuid as RamseyUuid;
+use Ramsey\Uuid\Exception\UnsatisfiedDependencyException;
 
 trait Uuid
 {
+    protected $uuid_version = 4;
```

Comment:
This will not work : you can't override and attribute defined in a trait from your class, you will get a fatal error from PHP.
Using a getter function for both these attributes (`getUuidVersion()` and `getUuidString()`), and override them in your class is probably the best solution.